articles/quickstart/backend/django/01-authorization.md

```diff
@@ -0,0 +1,251 @@
+
+Obtain the public key from your [JWKS](/jwks). Then set the settings for [REST Framework JWK](http://getblimp.github.io/django-rest-framework-jwt/).
+
+Set the `JWT_AUDIENCE` to your API identifier and the `JWT_ISSUER` to your Auth0 domain. By default those values will be retrieved from the `.env` file.
+
+```python
+# apiexample/settings.py
+
+jsonurl = request.urlopen("https://${account.namespace}/.well-known/jwks.json")
+jwks = json.loads(jsonurl.read())
+cert = '-----BEGIN CERTIFICATE-----\n' + jwks['keys'][0]['x5c'][0] + '\n-----END CERTIFICATE-----'
+
+certificate = load_pem_x509_certificate(str.encode(cert), default_backend())
+publickey = certificate.public_key()
+
+JWT_AUTH = {
+    'JWT_PAYLOAD_GET_USERNAME_HANDLER':
+        'authorization.user.jwt_get_username_from_payload_handler',
+    'JWT_PUBLIC_KEY': publickey,
+    'JWT_ALGORITHM': 'RS256',
+    'JWT_AUDIENCE': '${apiIdentifier}',
+    'JWT_ISSUER': '${account.namespace}',
+    'JWT_AUTH_HEADER_PREFIX': 'Bearer',
+}
+```
+
+## Add a Django User
+
+You need to define a way to map the username from the `access_token` payload to the Django authentication system user.
+
+Create `user.py` file in your application's folder and define a function that maps the  `sub` field from the `access_token` to the username.
+
+```python
+# auth0authorization/user.py
+
+def jwt_get_username_from_payload_handler(payload):
+    return payload.get('sub')
+```
+
+Then create a user in Django authentication system. Please check the Django documentation [Django documentation](https://docs.djangoproject.com/en/1.11/topics/auth/default/#creating-users) for more information. 
+
+## Protect Individual Endpoints
+
+In the file `views.py` add `public` and `private` endpoints. Add the `@api_view` decorator to the `private` endpoint to indicate that the method requires authentication.
+
+```python
+# auth0authorization/views.py
+
+from functools import wraps
+
+from rest_framework.decorators import api_view
+from django.http import HttpResponse
+from jose import jwt
+
+def public(request):
+    return HttpResponse("All good. You don't need to be authenticated to call this")
+
+
+@api_view(['GET'])
+def private(request):
+    return HttpResponse("All good. You only get this message if you're authenticated")
+```
+
+## Configure the Scopes
+
+To configure scopes, click the Scopes section of the [Dashboard's APIs section](${manage_url}/#/apis) and configure the scopes you need.
+
+::: note
+This example uses the read:messages scopes.
+:::
+
+API endpoints can be configured to look for a particular `scope` in the `access_token`.
+
+Add the following methods to the `views.py` file to extract the granted scopes from the access_token.
```

Comment:
When using "access_token" should probably be consistent in how it's notated. I'd stick with `access_token` 
 in code syntax when referring to _the_ token you're currently working with, and "access token" when discussing access tokens in general. But either way.